lsp-ada.el

```diff
@@ -0,0 +1,73 @@
+;;; lsp-ada.el --- description -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020 E. Alexander Barbosa
+
+;; Author: E. Alexander Barbosa <elxbarbosa@outlook.com>
+;; Keywords: lsp, ada
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; LSP Clients for the Ada Programming Language
+
+;;; Code:
+
+(require 'lsp-mode)
+
+;;; Ada
+(defgroup lsp-ada nil
+  "LSP support for the Ada Programming Language, using the Ada Language Server."
+  :group 'lsp-mode
+  :tag "Language Server"
+  :version "7.1")
+
+(defcustom lsp-clients-ada-project-file "default.gpr"
```

Comment:
Like @yyoncho said it [here](https://github.com/emacs-lsp/lsp-mode/pull/2028#issuecomment-671714613), you should keep the same variable names.
I think you can keep this name but 'define-obsolete-variable-alias' like [here](https://github.com/emacs-lsp/lsp-mode/blob/master/lsp-mode.el#L96-L101) to deprecate those old names.

Reply:
And you can keep the "6.1" variable package-version, since that already exists, it was only renamed

Reply:
done and done.

oh, my bad. I did lsp-ada before reading that, forgot to undo it before pushing :)